azkaban-db/src/main/sql/create.ramp_rules.sql

```diff
@@ -1,13 +1,13 @@
 CREATE TABLE ramp_rules (
     rule_name VARCHAR(100) NOT NULL,
-    image_name VARCHAR(200) NOT NULL,
-    image_version VARCHAR (100) NOT NULL,
+    image_name VARCHAR(200),
+    image_version VARCHAR (100),
     owners VARCHAR (1000) NOT NULL,
     is_HP BIT NOT NULL,
     created_by VARCHAR(20) NOT NULL,
     created_on DATE NOT NULL,
-    modified_by VARCHAR(20),
-    modified_on DATE,
+    modified_by VARCHAR(20) NOT NULL,
+    modified_on DATE NOT NULL,
```

Comment:
Consider this

```suggestion
    modified_on DATE NOT NULL ON UPDATE CURRENT_TIMESTAMP,
```

Reply:
our mysql version is old, it does not support this syntax.